spikeinterface_gui/tracemapview.py

```diff
@@ -49,7 +49,8 @@ def apply_gain_zoom(self, factor_ratio):
     def auto_scale(self):
         if self.last_data_curves is not None:
             self.color_limit = np.max(np.abs(self.last_data_curves))
-        self.refresh()
+            self.color_mapper.high = self.color_limit
+            self.color_mapper.low = -self.color_limit
```

Comment:
No.
This is panel only. This should be done in panel refersh

Reply:
mmm not sure this is panel only. Why?